### PR TITLE
Development

### PR DESCRIPTION
**Pull request by solufemi91**
This is Steven's work for the first sprint

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      var $keeperPosition = $goalkeeper.position();
+
+      if($keeperPosition.left > 50 && $keeperPosition.left < 227){
+        alert("Oh what a fantastic save from the keeper!");
+        save++;
+        checkScore();
+        $keeperScore.html("Keeper: " + save);
+        $footballImage.css("top", ballDistance + 'px');
+      } else {
+        alert("Cool, carm and collected. You scored!");
```
Comment (JSChilds) [inline main.js]: This needs to be indented.

File: test.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
```
Comment (JSChilds) [inline test.html]: Remove this file from the file system if it's not being used.

**Review by JSChilds — CHANGES_REQUESTED** [review]
Make the changes highlighted in the comments. It also seems like the keeper can't actually save the goal, in which case there is no real challenge.

**Review by JSChilds — APPROVED** [review]

**Review by j-mcgregor — CHANGES_REQUESTED** [review]
You need to remove the alert() functions in the code.

**Review by j-mcgregor — APPROVED** [review]
Very nice, well done

**Review by j-mcgregor — APPROVED** [review]

**Review by richgurney — CHANGES_REQUESTED** [review]
There is a bug with the game when you score the keeper gets a point. Also you will need to rearrange the home page because at the moment the index.html will be loaded first and the home.html will not be seen.

**Review by richgurney — APPROVED** [review]

**Pull request merged**